draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.scss

```diff
@@ -40,7 +42,7 @@ $dt-tooltip-content-border: var(
 }
 
 .arrow {
-  z-index: 2;
+  z-index: $ca-z-index-tooltip;
```

Comment:
As for `.tooltipContent`, I don't believe the `z-index` here is required if we decide to have `.tooltip` create the stacking context for the entire tooltip. If `.arrow` should stack on top of `.tooltipContent`, then the automatic stacking order will do this without us meddling with `z-index` values. If `.tooltipContent` should stack on top of `.arrow`, then we only need to specify an elevated `z-index` property for `.tooltipContent`, and we can leave `.arrow` on the base layer.